esrally/rally.py

```diff
@@ -614,20 +621,33 @@ def race(cfg):
     other_rally_processes = process.find_all_other_rally_processes()
     if other_rally_processes:
         pids = [p.pid for p in other_rally_processes]
-        msg = "There are other Rally processes running on this machine (PIDs: %s) but only one Rally benchmark is allowed to run at " \
-              "the same time. Please check and terminate these processes and retry again." % pids
-        raise exceptions.RallyError(msg)
 
-    with_actor_system(racecontrol.run, cfg)
+        kill_running_processes = cfg.opts("system", "kill.running.processes", default_value=False, mandatory=False)
 
+        if kill_running_processes:
+            console.info("Killing running processes ...", flush=True)
+            for pid in pids:
```

Comment:
We already had this functionality prior to 8adb0f89097a13a27a6b64bb92255c01c2d058e0. Can you please check how it was done there and restore that functionality? I also don't recall that it was necessary to make the actor system bootstrap code aware of this so I'd try to avoid it (after restoring the functionality removed in 8adb0f89097a13a27a6b64bb92255c01c2d058e0).

Reply:
Sure! I restored the previous functionality and avoided changes in the actor system bootstrap code. Thanks for your suggestion. Can you check again if something needs to be changed?